adobe_vipm/flows/fulfillment/change.py

```diff
@@ -288,31 +289,36 @@ def fulfill_change_order(mpt_client, order):
 
     if not adobe_order:
         return
+
+
+    one_time_skus = get_one_time_skus(mpt_client, order)
     updated_lines = []
+
     for item in adobe_order["lineItems"]:
         order_line = get_order_line(
             order,
             item["extLineItemNumber"],
         )
         updated_lines.append(order_line)
-        order_subscription = get_subscription_by_line_and_item_id(
-            order["subscriptions"],
-            order_line["item"]["id"],
-            order_line["id"],
-        )
-        if not order_subscription:
-            add_subscription(
-                mpt_client, adobe_client, customer_id, order, ITEM_TYPE_ORDER_LINE, item
+        if item["offerId"][:10] not in one_time_skus:
```

Comment:
I would recommend here to add a separate function to extract basic sku. Similar code is already exists in 5 places and i think it is time to extract it and avoid +1 place :-) Agree?

![CleanShot 2024-06-05 at 11 18 57](https://github.com/softwareone-platform/swo-adobe-vipm-extension/assets/2346552/37a204b5-1199-4c51-80fa-cf09f065cf69)